examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
Proj operates spatially and filtering operates temporally so it shouldn't
break like this. The correct order is to apply proj then pick, not pick
then apply partial proj (as we do in master; this is a bug). I suspect
there must be some problem with our apply_proj method or the picking
afterward. Do the logs say projection is actually being applied?